### ISSUE TITLE
Make Loader component accesible

### ISSUE THREAD

**Issue by meelrossi** [decentraland/ui#310]
## 🎉 Description

Make loader accesible so the user can understand that some info is being loaded in the screen
Move test utils in the marketplace to use the loader accesible properties instead of classname

 ```
export async function waitForComponentToFinishLoading(screen: RenderResult) {
  // TODO: Make loader accessible so we can get the info without using the container
  await waitFor(() =>
    expect(screen.container.getElementsByClassName('loader-container').length).toEqual(
      0
    )
  )
}
```
* [ ] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

## 📝 Details

Describe the problem you have been experiencing in more detail. Include as much information as you think is relevant. Keep in mind that transactions can fail for many reasons; context is key here. Links to tx, screenshots are welcome.
Remember to not share sensible information

## 🔢 Steps To Reproduce Issue [ Good To Have ]

Please remember that with baby steps it's easier to reproduce the bug and it's much faster to fix it.

```
- Step 1
- Step 2
- Step 3
...
```
